2017-10/announcements/announcements.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Ann Arbor R User Group</title>
+    <meta charset="utf-8">
+    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
+    <link rel="stylesheet" href="static/style.css" type="text/css" />
+  </head>
+  <body>
+    <textarea id="source">
+class: center, middle, inverse, title-slide
+
+# Ann Arbor R User Group
+## October 12, 2017
+
+---
+
+
+.pull-left[
+## Group Info
+
+- [AnnArborRUserGroup.github.io](https://AnnArborRUserGroup.github.io)
+
+- [meetup.com/Ann-Arbor-R-User-Group](http://meetup.com/Ann-Arbor-R-User-Group)
+
+- [AnnArborRUserGroup@gmail.com](AnnArborRUserGroup@gmail.com)
+]
+
+.pull-right[
+## Organizers
+
+![](https://avatars3.githubusercontent.com/u/9932219?v=3&amp;s=96) Ellis
+
+![](https://avatars3.githubusercontent.com/u/5169330?v=3&amp;s=96) Clayton
+
+![](https://avatars3.githubusercontent.com/u/5975297?v=3&amp;s=96) Rob
+
+![](https://avatars3.githubusercontent.com/u/8618665?v=3&amp;s=96) Andrew
+]
+
+---
+class: center, middle
+
+&lt;img src="static/slack_cmyk.png" width="200"&gt;
+
+&lt;img src="static/everywhere@2x.png" width="200"&gt;
+
+### [https://a2mads.herokuapp.com/](https://a2mads.herokuapp.com/)
+
+---
+class: center, middle
+
+# Sponsors
+
+&lt;img src="static/RConsortium_Horizontal_Pantone.png"  style="max-width: 30%; display: block; margin: 0 auto;"&gt;
+
+&lt;img src="static/spark-logo.jpeg" style="max-width: 30%; display: block; margin: 0 auto;"&gt;
+
+---
+class: center, middle
+
+# Future AARUG Meetups
+
+### November 9 - 6:30 PM @ SPARK
+#### `spotifyr` / package-development and...you?
+
+### December - social TBD
+
+---
+class: center, middle
+
+# Other Events
+
+### PyData Ann Arbor - Tuesday, November 14 @ TD Ameritrade
+
+### a2-dlearn - November 17, 2017 @ Bob and Betty Beyster Building
+
+---
+class: center, middle
+
+# This month in R
```

Comment:
Technically not _this_ month but R 3.4.2 (Short Summer) was released since our last meetup.